docker-compose.yml

```diff
@@ -13,6 +13,8 @@ crossdock:
 
       - BEHAVIOR_DANCE=client,server,transport
       - BEHAVIOR_SING=client,server,transport
+      - BEHAVIOR_RUN=client,server,transport
+      - BEHAVIOR_SKIP_RUN=client=alpha+server=alpha
```

Comment:
This might be more legible with colons and commas. We already use comma as a delimiter, and `=` for the assignment.

```
SKIP_RUN=client:alpha,server:alpha
```